apps/src/templates/sectionProgress/progressTables/progressTableHelpers.js

```diff
@@ -1,14 +1,106 @@
+import React from 'react';
 import moment from 'moment';
+import {lessonIsAllAssessment} from '@cdo/apps/templates/progress/progressHelpers';
+import ProgressTableSummaryCell from './ProgressTableSummaryCell';
+import ProgressTableDetailCell from './ProgressTableDetailCell';
+import ProgressTableLevelSpacer from './ProgressTableLevelSpacer';
+import ProgressTableLevelIconSet from './ProgressTableLevelIconSet';
+import * as progressStyles from '@cdo/apps/templates/progress/progressStyles';
 
-export function timeSpentFormatter(studentProgress) {
+// getSummaryCellFormatters returns the cell formatters for the progress table summary view
```

Comment:
NIT: imo we should use block-style comments for functions, classes, etc.

that said, good work documenting this functionality! :)

Reply:
Yes! I actually updated this locally just haven't pushed it up yet!

Reply:
🙌